Eevee.Sleep.Bot/Models/InGameAnnouncementIndexModel.cs

```diff
@@ -0,0 +1,7 @@
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Eevee.Sleep.Bot.Models;
+
+[BsonIgnoreExtraElements]
+public record InGameAnnouncementIndexModel: InGameAnnouncementMetaModel {
```

Comment:
See if empty brackets can be omitted in some way. If not. make empty brackets to be in the same line.